tests/e2e/workloads/amq/test_amq_streamer_creation.py

```diff
@@ -46,15 +46,11 @@ def test_install_amq_cephfs(self, interface, test_fixture_amq):
         # Deploy amq cluster
         test_fixture_amq.setup_amq_cluster(sc.name)
 
-        # Run open messages
-        test_fixture_amq.create_messaging_on_amq()
-
-        # Wait for some time to generate msg
-        waiting_time = 60
-        log.info(f"Waiting for {waiting_time}sec to generate msg")
-        time.sleep(waiting_time)
-
-        # Check messages are sent and received
-        threads = test_fixture_amq.run_in_bg()
-        for t in threads:
-            t.join()
+        # Run benchmark
```

Comment:
As per https://docs.google.com/presentation/d/15WV0AmtTy58G2JRM8fAfYM6hVDl3oljrg8eKEswKHBk/edit#slide=id.g70daec23bd_0_312, AMQ is recommended on CephFS. Better to ignore running on cephfs for now.

Reply:
Updated a PR to run amq on rbd only